bento_beacon/authz/headers.py

```diff
@@ -2,6 +2,9 @@
 
 
 def auth_header_getter(r: Request) -> dict[str, str]:
```

Comment:
this one is fiddly.. r is not actually a request, it's a werkzeug [LocalProxy](https://werkzeug.palletsprojects.com/en/2.3.x/local/) that acts like a request when a request happens to present. When no request is present it's just `LocalProxy` rather than none.

Reply:
hmm...

Reply:
I'm considering just removing this and handling the no-request edge case elsewhere

Reply:
removed